components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/gen/java/org/wso2/carbon/identity/api/server/idp/v1/model/JustInTimeProvisioning.java

```diff
@@ -68,6 +68,40 @@ public static SchemeEnum fromValue(String value) {
     private String userstore = "PRIMARY";
     private Boolean associateLocalUser = false;
 
+@XmlType(name="AttributeSyncMethodEnum")
+@XmlEnum(String.class)
+public enum AttributeSyncMethodEnum {
+
+    @XmlEnumValue("OVERRIDE_ALL") OVERRIDE_ALL(String.valueOf("OVERRIDE_ALL")), @XmlEnumValue("NONE") NONE(String.valueOf("NONE")), @XmlEnumValue("PRESERVE_LOCAL") PRESERVE_LOCAL(String.valueOf("PRESERVE_LOCAL"));
+
+
+    private String value;
+
+    AttributeSyncMethodEnum(String v) {
+        value = v;
+    }
+
+    public String value() {
+        return value;
+    }
+
+    @Override
+    public String toString() {
+        return String.valueOf(value);
+    }
+
+    public static AttributeSyncMethodEnum fromValue(String value) {
+        for (AttributeSyncMethodEnum b : AttributeSyncMethodEnum.values()) {
```

Comment:
formatting issue.missing a new line after the method signature. Check other places as well

Reply:
this is a auto generated file. Do we need to manually format them?